assets/package.json

```diff
@@ -2,17 +2,13 @@
   "repository": {},
   "description": " ",
   "license": "MIT",
-  "scripts": {
-    "deploy": "webpack --mode production",
-    "watch": "webpack --mode development --watch"
-  },
+  "scripts": {},
```

Comment:
@ngscheurich Any thoughts on how to write a script here that mirrors `mix lint` I added to combine all the JS linters/formatters we have in CI into one command to run locally?